.github/actions/test-linux/action.yml

```diff
@@ -30,4 +30,4 @@ runs:
           --offline \
           --show-diff \
           --show-slow 1000 \
-          --set-timeout 120
+          --set-timeout 1200
```

Comment:
https://github.com/php/php-src/runs/8296934151?check_suite_focus=true#step:11:3157 shows when the issue is hit, there is no OCI8 timeout or the timeout is higher than 10 minutes.

Reply:
based on:
https://github.com/mvorisek/php-src/runs/8302900173?check_suite_focus=true#step:15:108
https://github.com/mvorisek/php-src/runs/8302902211?check_suite_focus=true#step:10:117
(with elapsed time dump)

it seems "Test 7" - "Test 10" can take up ~200 seconds / "Test XX"